rust/opendp/src/trans/dataframe/mod.rs

```diff
@@ -81,49 +80,17 @@ pub fn make_split_dataframe<K>(
         StabilityRelation::new_from_constant(1)))
 }
 
-fn replace_col<K: Eq + Hash + Debug + Clone>(key: &K, df: &DataFrame<K>, col: Column) -> Fallible<DataFrame<K>> {
-    let mut df = df.clone();
-    *df.get_mut(key)
-        .ok_or_else(|| err!(FailedFunction, "column does not exist: {:?}", key))? = col;
-    Ok(df)
-}
-
-fn parse_column<K, T>(key: &K, impute: bool, df: &DataFrame<K>) -> Fallible<DataFrame<K>>
-    where T: 'static + Debug + Clone + PartialEq + FromStr + Default,
-          K: Eq + Hash + Clone + Debug,
-          T::Err: Debug {
-    let col: &Vec<String> = df.get(key)
-        .ok_or_else(|| err!(FailedFunction, "column does not exist: {:?}", key))?
-        .as_form()?;
-    let col = vec_string_to_str(col);
-    let col = parse_series::<T>(&col, impute)?;
-    replace_col(key, &df, col.into())
-}
```

Comment:
Awww... I was so proud of myself when I figured out how to implement this. 😂

Reply:
I'm sorry! But the lessons learnt are not forgotten. We'll be using this more in the future.